hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/operations/Backup.java

```diff
@@ -100,19 +101,29 @@ public void beforeRun() throws Exception {
         PartitionReplicaVersionManager versionManager = partitionService.getPartitionReplicaVersionManager();
         namespace = versionManager.getServiceNamespace(backupOp);
 
+        if (!nodeEngine.getNode().getNodeExtension().isStartCompleted()) {
+            valid = false;
```

Comment:
Does this get invoked in the `run` method then? 
```
onExecutionFailure(new IllegalStateException("Wrong target! " + toString() + " cannot be processed!"));
```

Reply:
hmm, good point. I'll fix that.